runtime-acceptance-tests/src/main/java/org/activiti/cloud/qa/helpers/ProcessDefinitionRegistry.java

```diff
@@ -13,13 +13,14 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
+
 package org.activiti.cloud.qa.helpers;
 
 import java.util.HashMap;
 
 public class ProcessDefinitionRegistry {
 
-    private static final HashMap <String, String> processWithTasksDefinitionKeys = new HashMap<>() {{
+    private static final HashMap<String, String> processWithTasksDefinitionKeys = new HashMap<String, String>() {{
```

Comment:
Why do we need this change? If the compiler is complaining about it, we are probably missing to set the java version. `new HashMap<>()` is valid since Java 7.

Reply:
Actually, the thing here is the double brace initialisation, which creates an anonymous inner class -> in general we should avoid these, but it needs the types for compiling as it is an anonymous inner class, not a general collection which then infers the type

Reply:
I missed the `{{`. I agree that we shouldn't be using that but it's out of the scope of this PR. Let's fixed it later on.